x/bank/client/cli/cli_test.go

```diff
@@ -108,7 +112,7 @@ func (s *IntegrationTestSuite) TestGetBalancesCmd() {
 				s.Require().Error(err)
 			} else {
 				s.Require().NoError(err)
-				s.Require().NoError(val.ClientCtx.LegacyAmino.UnmarshalJSON(out.Bytes(), tc.respType))
+				s.Require().NoError(val.ClientCtx.JSONMarshaler.UnmarshalJSON(out.Bytes(), tc.respType.(proto.Message)))
```

Comment:
Instead of casting here, let's just change `respType` from `fmt.Stringer` -> `proto.Message`. I'm not sure why we're using `fmt.Stringer` everywhere but it really should be `proto.Message`